<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Kamera</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtSoyad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Soyadı</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtAd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Adı</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFotoCek">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Fotoğraf Çek</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTamam">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamam</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>330</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adı :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>370</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Soyadı :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>410</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kurum :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>450</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kimlik :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtKurum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kurum</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtKimlik">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kimlik</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Kamera   1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kamera   2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kamera   3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kamera   4</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
